heutagogy/views.py

```diff
@@ -55,7 +55,11 @@ def get(self):
         filters = [db.Bookmark.user == current_user.id]
         if url is not None:
             filters.append(db.Bookmark.url == urldefrag(url).url)
-        filters.append(db.Bookmark.tags.contains(tags))
+
+        # If Bookmark.url is null, filtering will yield no results
+        if tags:
```

Comment:
Having a strict predicate makes the code more understable and less error-prone: 

Compare js code:
• `if (condition)` vs `if (condition === true)`
• `if (condition)` vs `if (condition === undefined)`
• `if (condition.length)` vs `if (condition.length > 0)`

Reply:
I kind of agree. Though, afaik, PEP8 recommends this style.

Basically, this line is equivalent to `it tags is not None and len(tags) > 0:`